controllers/validate.go

```diff
@@ -0,0 +1,83 @@
+// Package controllers is the layer for imput date in the application.
```

Comment:
Please add the content of this file in `docker.go` (this is docker specific thing)

Reply:
okay